crates/bin/pd/src/main.rs

```diff
@@ -13,7 +13,7 @@ use cnidarium::{StateDelta, Storage};
 use metrics_exporter_prometheus::PrometheusBuilder;
 use pd::{
     cli::{Opt, RootCommand, TestnetCommand},
-    migrate::Migration::SimpleMigration,
+    migrate::Migration::Testnet70,
```

Comment:
Nit: should this be Testnet71? That would make it clearer that we expect it to ship with `v0.71.0`. It's not exposed to operators, so we just need to keep the terminology straight ourselves.

Reply:
SG, my reasoning was 70 -> 71, but the other way around works too